client/extensions/woocommerce/app/store-stats/index.js

```diff
@@ -36,8 +43,8 @@ class StoreStats extends Component {
 				<Navigation unit={ unit } type="orders" slug={ slug } />
 				<Chart
 					path={ path }
-					query={ ordersQuery }
-					selectedDate={ selectedDate }
+					query={ Object.assign( {}, ordersQuery, { date: unitQueryDate } ) }
```

Comment:
Why create a new object here. Is there a reason to not simply modify the original `ordersQuery`? The query is used by the selectors to grab the correct data. Here, its being done by the `Chart` anyways so it won't matter. But staying consistent may prevent future errors.